pkg/test/main_entry.go

```diff
@@ -25,8 +25,22 @@ import (
 func MainEntry(m *testing.M) {
 	fopts := &frameworkOpts{}
 	fopts.addToFlagSet(flag.CommandLine)
+	// controller-runtime registers the --kubeconfig flag in client config
+	// package:
+	// https://github.com/kubernetes-sigs/controller-runtime/blob/v0.5.2/pkg/client/config/config.go#L39
```

Comment:
For we use the 0.5.2 we need also, upgrade the controller-runtime version.

Reply:
I think the point of Eric's comment is that in `client/config/config.go` the flag is being registered. I don't think the version is actually relevant because the same flag registration happens in both `0.4.0` which we have in `go.mod` and in `v0.5.2` which is the latest `controller-runtime` release. 

I would not update controller-runtime for this change and I would not change the comment.

Reply:
I have another pr already to update it.
Yep, the flag has in the current version as well.